v1/pdf.yaml

```diff
@@ -12,7 +12,7 @@ info:
     name: Apache licence, v2
     url: https://www.apache.org/licenses/LICENSE-2.0
 paths:
-  /pdf/{title}:
+  /pdf/{title}/{format}/{type}:
```

Comment:
We cant really do this, this is not backwards compatible so we will break all the existing clients. Both format and type should be optional, this this needs to be split into 3 separate routes. Also, need to document the defaults for the optional parameters.